components/BottomNavigation/tests/snapshot/MDCBottomNavigationBarControllerSnapshotTests.m

```diff
@@ -0,0 +1,230 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialSnapshot.h"
+
+#import "MDCBottomNavigationBarController.h"
+
+/** Number of cells shown in the scrollable child view controller. */
+static const NSInteger kNumberOfCellsInScrollableChild = 25;
+
+/** The size of a cell in the scrollable child view controller. */
+static const CGSize kSizeOfCellInScrollableChild = (CGSize){(CGFloat)48, (CGFloat)48};
```

Comment:
Tip: `CGSize kSizeOfCell.. = {48, 48};`

Reply:
I'll admit I definitely don't like creating static `CGSize` structs like this. I would be even more worried about the *implicit* casting of two `int_32t` values onto `double` values for 64-bit architectures. I'm doing an explicit cast here to ensure that the widths of each value in the right hand side will be correct for the architecture.

If you can point me to docs that reassure me that the compiler will do the right thing without explicit casts, I'd be happy to remove them.